src/store/getters.ts

```diff
@@ -15,7 +15,7 @@ export default {
 		};
 	},
 	property( rootState: RootState ): Property | null {
-		if ( !rootState.conditionRow.propertyData.id ) {
+		if ( !rootState.conditionRow.propertyData || !rootState.conditionRow.propertyData.id ) {
```

Comment:
I think we can write this nicer with optional chaining: `?.`

See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#optional-chaining

Reply:
Ooooh thanks. Changed.